src/mumble/ListenerVolumeSlider.h

```diff
@@ -20,12 +23,20 @@ class ListenerVolumeSlider : public VolumeSliderWidgetAction {
 	void setListenedChannel(const Channel &channel);
 
 private:
-	/// The channel of the listener proxy this dialog is operating on
+	/// The channel of the listener proxy this widget is operating on
 	const Channel *m_channel;
 
+	QTimer m_sendTimer;
+	QTimer m_resetTimer;
+	unsigned int m_currentTimer;
+	int m_channelBuffer;
+	VolumeAdjustment m_adjustmentBuffer;
```

Comment:
When I read "buffer" I always have something like a vector (some kind of sequence that is) in mind. Maybe "cache" is the better term?
```suggestion
	int m_cachedChannelID;
	VolumeAdjustment m_cachedAdjustment;
```